rails/app/controllers/developments_controller.rb

```diff
@@ -55,7 +55,10 @@ def create
         format.jsonapi { render json: FullDevelopmentSerializer.new(@development).serialized_json }
       end
     else
-      render json: FullDevelopmentSerializer.new(@development.errors.full_messages).serialized_json, status: :bad_request
+      puts @development.errors.full_messages
```

Comment:
Was this for temporary debugging? Could we still find a way to respond with the error to the client and not simply a `head :bad_request`?